excel.test.ts

```diff
@@ -96,10 +96,43 @@ describe('Excel.toHtml() end usage tests', () => {
         expect(excel.toHtml(`EQ(person, ['name', 'lastname', 'age'])`)).toContain(expectedStringArray);
     });
 
+    test('Testing parsing objects', () => {
+        expect(excel.toHtml(`HTTP({method: 'GET', url: 'https://api.github.com/users/defunkt'})`))
+            .toEqual(removeExtraSpaces(`
+                <div>
+                    <span class="function">HTTP</span>
+                    <span class="paren-deep-1">(</span>
+                    {
+                        method: <span class="value">'GET'</span>, 
+                        url: <span class="value">'https://api.github.com/users/defunkt'</span>
+                    }
+                    <span class="paren-deep-1">)</span>
+                </div>
+            `));
+    });
+
+    test('Testing parsing nested objects', () => {
+        expect(excel.toHtml(`HTTP({method: 'GET', url: 'https://api.github.com/users/defunkt', headers: { 'User-Agent': 'request' }})`))
+          .toEqual(removeExtraSpaces(`
+                <div>
+                    <span class="function">HTTP</span>
+                    <span class="paren-deep-1">(</span>
+                    {
+                        method: <span class="value">'GET'</span>, 
+                        url: <span class="value">'https://api.github.com/users/defunkt'</span>,
+                        headers: {
+                            User-Agent: <span class="value">'request'</span>
```

Comment:
And here User-Agent is not quoted. Can we make it consistent?

Reply:
In JS, object keys with hyphens should be in quotes. But here in a formula, we probably can go without quotes.

Reply:
fixed!, now any combination of these:
```
excel.toHtml(`HTTP( { method: 'GET' } )
excel.toHtml(`HTTP( { 'method': 'GET' } )
excel.toHtml(`HTTP( { "method": 'GET' } )
```
Produce the same HTML output.